<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcStarAppClass</class>
 <widget class="QMainWindow" name="CalcStarAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FrmMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabCtrlMain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>471</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_Variables">
     <attribute name="title">
      <string>VARIABLES</string>
     </attribute>
     <widget class="QTableWidget" name="grd_Variables">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>421</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_Variables">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Variables</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cmd_SaveVariables">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Variables</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Tokens">
     <attribute name="title">
      <string>TOKENS</string>
     </attribute>
     <widget class="QLabel" name="lbl_Tokens">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tokens List</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="grd_Tokens">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>421</width>
        <height>361</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_RPN">
     <attribute name="title">
      <string>RPN</string>
     </attribute>
     <widget class="QLabel" name="lbl_RPN">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RPN Stack List</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="grd_RPNStack">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>31</y>
        <width>421</width>
        <height>361</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Output">
     <attribute name="title">
      <string>OUTPUT</string>
     </attribute>
     <widget class="QLabel" name="lbl_Output">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Expression Parser Output</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="txt_Output">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabTests">
     <attribute name="title">
      <string>TEST RESULTS</string>
     </attribute>
     <widget class="QTextEdit" name="txtTestResults">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="txt_Expression">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_strExpression">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>strExpression</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="mnu_Expression">
    <property name="title">
     <string>Expression</string>
    </property>
    <addaction name="actionTokenize"/>
    <addaction name="actionCompile"/>
    <addaction name="actionEvaluate"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionRunAllTests"/>
   </widget>
   <addaction name="mnu_File"/>
   <addaction name="mnu_Expression"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTokenize">
   <property name="text">
    <string>Tokenize</string>
   </property>
  </action>
  <action name="actionCompile">
   <property name="text">
    <string>Compile</string>
   </property>
  </action>
  <action name="actionEvaluate">
   <property name="text">
    <string>Evaluate</string>
   </property>
  </action>
  <action name="actionRunAllTests">
   <property name="text">
    <string>Run All Tests</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CalcStarApp.qrc"/>
 </resources>
 <connections/>
</ui>
